### DISCUSSION TITLE
Still under active development?

### DISCUSSION THREAD

**Discussion by uurf** [mriscoc/Ender3V2S1#1362] [General]
I've been very appreciative of this work. The release cadence seems to have slowed down (5 releases in 2023, 1 so far in 2024) - is this project still under active development?

Maybe the last release was so good it hasn't needed any updates. :)

**Comment by mriscoc** [comment]
There was an update only three weeks ago. Check the GitHub.

https://github.com/mriscoc/Ender3V2S1/commits/Ender3V2S1-Released/

**Reply by uurf** [reply]
I had, looking [here](https://github.com/mriscoc/Ender3V2S1/releases). :)

**Reply by uurf** [reply]
It looks like there's been [13 commits since the last release](https://github.com/mriscoc/Ender3V2S1/releases), do you anticipate cutting a new release to contain those?

**Reply by bjoernbusch** [reply]
There used to be actual releases quite regularly, but in 2024 there is just one https://github.com/mriscoc/Ender3V2S1/releases

**Reply by uurf** [reply]
@mriscoc It looks like there's been [13 commits since the last release](https://github.com/mriscoc/Ender3V2S1/releases), do you anticipate cutting a new release to contain those commits?

If not, is there documentation to build the required binaries from the current branch?

**Reply by mriscoc** [reply]
Look here: https://github.com/mriscoc/Special_Configurations#creating-your-own-custom-configuration
